docs/reference/commandline/plugin_create.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "plugin create"
+description: "the plugin create command description and usage"
+keywords: ["plugin, create"]
+advisory: "experimental"
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# plugin create (experimental)
```

Comment:
Fixed. The last 3 comments apply across all plugin md files.  @thaJeztah, is there a tool to run to check if the md files confirm to the new format? I ask because there might be a lot more across docker/docker.